packages/driver/src/multi-domain/index.ts

```diff
@@ -239,17 +244,10 @@ const onBeforeAppWindowLoad = (cy: $Cy, Cypress: Cypress.Cypress) => (autWindow:
       return ret
     },
   })
-
-  specBridgeCommunicator.toPrimary('window:before:load')
 }
 
 // eventually, setup will get called again on rerun and cy will
 // get re-created
-const cy = setup()
-
-// @ts-ignore
-window.__onBeforeAppWindowLoad = (autWindow: Window) => {
-  cy.onBeforeAppWindowLoad(autWindow)
```

Comment:
this is now executed via `Cypress.action('app:window:before:load', window)` in multi-domain.js which also emits the `window:before:load` event